wbooks/src/app/screens/dashboard/screens/Detail/components/CommentList/components/Comment/index.js

```diff
@@ -26,16 +29,12 @@ const Comment = ({ user, image_url, date, content }) => {
 };
 
 Comment.defaultProps = {
-  user: 'Kimberly Carter',
   image_url: userImgPng,
   date: 'xx/xx/xx',
   content: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.'
 };
 
 Comment.propTypes = {
-  image_url: PropTypes.string,
-  user: PropTypes.string.isRequired,
-  date: PropTypes.string.isRequired,
```

Comment:
Remember to add prop validations to these that are missing. By example, `user` shape is missing props validations